classes/profile_table.php

```diff
@@ -146,4 +155,25 @@ public function col_user(object $record): string {
             }
         }
     }
+
+    /**
+     * Display for the delete column
+     *
+     * @param object $record
+     * @return mixed
+     * @throws \coding_exception
+     * @throws \moodle_exception
+     */
+    public function col_delete(object $record) {
+        global $OUTPUT;
+        $deleteurl = new \moodle_url('/admin/tool/excimer/delete.php', [ 'deleteid' => $record->id, 'sesskey' => sesskey() ]);
```

Comment:
```suggestion
        $deleteurl = new \moodle_url('/admin/tool/excimer/delete.php', ['deleteid' => $record->id, 'sesskey' => sesskey()]);
```
Please kill the surrounding whitespace